### PR TITLE
feat: Add Github Action cache and doc URL checker 

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

Added URL checker for:
- Balsamic cache links
- Documentation

### Review and tests: 
- [x] Tests pass
  Screenshot 2023-08-15 at 16 25 22

- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: .github/workflows/urls_check.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Check URLs
+on:
+  push:
+    branches:
+      - master
+      - develop
+jobs:
+  main:
+    name: Find broken cache and documentation links
```
Comment (fevac) [inline .github/workflows/urls_check.yml:9]: Love it!! ❤️

File: .lycheeignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+https:\/\/gatk\.broadinstitute\.org\/hc\/.*
```
Comment (fevac) [inline .lycheeignore:1]: why are we ignoring these?

**Review by fevac — APPROVED** [review]
AMAZING! great action 🌟 🪨 !!

**Review by fevac — COMMENTED** [review]
update the changelog?

File: .lycheeignore (hunk 2 above)
Comment (ivadym) [reply .lycheeignore:1]: Their servers are denying any request we make it seems:

> Failed: Network error: Forbidden

**Comment by fevac** [comment]
> update the changelog?

Just to make sure I understand:
- so we will go ahead and merge this to master without updating the changelog (bypassing protections) as this is not affecting balsamic and only the git actions. There will be no version bump for this commit.

Am I right?

**Comment by ivadym** [comment]
> > update the changelog?
> 
> Just to make sure I understand:
> 
> * so we will go ahead and merge this to master without updating the changelog (bypassing protections) as this is not affecting balsamic and only the git actions. There will be no version bump for this commit.
> 
> Am I right?

Excatly, same as we did in the past with other GitHub actions. The other branch will be merged to develop :)

**Pull request merged**